Makefile

```diff
@@ -24,6 +24,8 @@
 
 VERSION = $(shell git name-rev --name-only --tags --no-undefined HEAD 2>/dev/null || echo git-`git rev-parse --short HEAD`)
 MOD_ID = $(shell cat user.config mod.config 2> /dev/null | awk -F= '/MOD_ID/ { print $$2; exit }')
+MOD_VERSION = $(shell cat user.config $(MANIFEST_PATH) 2> /dev/null | awk -F':' '/Version/ { gsub(/[[:blank:]]/, "", $$2); print $$2; exit }')
+OVERWRITE_VERSION = $(shell cat user.config mod.config 2> /dev/null | awk -F= '/PACKAGING_OVERWRITE_MOD_VERSION/ { print $$2; exit }')
```

Comment:
Could rename this to OVERWRITE_MOD_VERSION.

Reply:
Yes, please do.

Reply:
Done.